scripts/capture.py

```diff
@@ -49,6 +51,9 @@ def capture_and_save(warmup, threshold, path, image):
         current_date = now.strftime(DIR)
         current_time = now.strftime(FILE)
 
+        if rotate:
+            frame = cv2.flip(frame, -1)
```

Comment:
I see what you did there.

What I would really like us to have though is the ability to rotate the input image by a given number of degrees around its center. Just like [here](https://stackoverflow.com/questions/9041681/opencv-python-rotate-image-by-x-degrees-around-specific-point#9042907).